src/inquirer/render/console/__init__.py

```diff
@@ -37,7 +37,7 @@ def render(self, question, answers=None):
         try:
             return self._event_loop(render)
         finally:
-            print("")
+            print('', end='')
```

Comment:
this is a nonsensical operation, use this instead
```suggestion
            pass
```

Reply:
It prints an empty string without a newline hh

Reply:
It print an empty string without a newline hh